--- lightbox-2.04/lightbox/js/lightbox.js~	2008-03-10 06:57:43.000000000 +0200
+++ lightbox-2.04/lightbox/js/lightbox.js	2008-09-06 18:18:50.245850157 +0300
@@ -46,8 +46,8 @@
 //  Configurationl
 //
 LightboxOptions = Object.extend({
-    fileLoadingImage:        'images/loading.gif',     
-    fileBottomNavCloseImage: 'images/closelabel.gif',
+    fileLoadingImage:        '/lightbox/images/loading.gif',
+    fileBottomNavCloseImage: '/lightbox/images/closelabel.gif',
 
     overlayOpacity: 0.8,   // controls transparency of shadow overlay
 
